addons/controllers/vspherecpiconfig_controller_test.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	clusterNamespace            = "default"
+	defaultString               = "default"
```

Comment:
This has to be done because this package is in the controllers package scope, and lint complains about it being the string "default" floating around and a constant which is = "default". Also clusterNamespace is a variable used everywhere in this package, probably don't want to create a constant with that name at package level.